src/app/ui/components/select/select.stories.tsx

```diff
@@ -7,10 +8,17 @@ import { SelectSectionLabel } from './select-section-label';
 
 const items: ListItemType[] = [{ label: 'Label 1' }, { label: 'Label 2' }];
 
-// Example implementation - remove with Storybook
-// ts-unused-exports:disable-next-line
-export function SelectExample() {
-  return (
+const meta: Meta<typeof Select> = {
+  component: Select,
+  tags: ['autodocs'],
+  title: 'Select',
+};
+
+export default meta;
+type Story = StoryObj<typeof Select>;
+
+export const Primary: Story = {
```

Comment:
If you import `Select` as an alias, then you will be able to name your story as `Select` and it will show up in the UI that way instead of as `Primary` 

`import { Select as Component } from './select';`

![Screenshot 2024-01-08 at 06 31 28](https://github.com/leather-wallet/extension/assets/2938440/2b0f959b-4c6d-4c18-88a9-c8ec6c212418)